Assets/MixedRealityToolkit.Services/BoundarySystem/MixedRealityBoundarySystem.cs

```diff
@@ -570,7 +590,8 @@ public bool Contains(Vector3 location, Boundary.Type boundaryType = Boundary.Typ
             }
 
             // Handle the user teleporting (boundary moves with them).
-            location = MixedRealityToolkit.Instance.MixedRealityPlayspace.InverseTransformPoint(location);
+            // todo: update...
```

Comment:
Can this TODO be clarified a little (i.e. what to update, how to update, what's the impact of not doing this?)

Reply:
And I think same for the todo down below.

Reply:
this was left over from an earlier round of edits. i'll remove :)